ext/opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -291,6 +291,10 @@ def _extract_events(events: Sequence[Event]) -> ProtoSpan.TimeEvents:
     )
 
 
+def _strip_characters(ot_version):
+    return "".join(filter(lambda x: x.isdigit() or x == ".", ot_version))
```

Comment:
Could also be implemented as:

```python
from re import sub

sub(r"[^\d.], "", ot_version)
```

Reply:
I feel like the current way is a bit more readable, regex is a bit too compact.